### PR TITLE
dynv6 provider

### PR DESCRIPTION
**Pull request by sstamoulis**
This is PR for #26. In order to run tests you will need to provide a token for authorization using the -token flag.

### CODE REVIEW

**Comment by sstamoulis** [comment]
This library now works fine using caddy on a dev server

File: provider.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	"fmt"
+
+	"github.com/libdns/libdns"
+)
+
+// Provider for dynv6 HTTP REST API
+type Provider struct {
+	// Token is required for authorization.
+	// You can generate one at: https://dynv6.com/keys
+	Token string
```
Comment (mholt) [inline provider.go:14]: Add JSON tag:
```suggestion
	Token string `json:"token,omitempty"
```
Allowing for easy JSON configuration :+1:

**Review by mholt — APPROVED** [review]
Cool, looks pretty good -- just a nit about JSON struct tag for configuration. It's great that it has tests!

I'll make a repo for it, and a caddy-dns repo as well if you would be able to push the DNS module there. :)

Make sure to apply a compatible OSS license when you push!

Thanks for making this!

**Comment by sstamoulis** [comment]
Hey ,thanks for the suggestions. Already updated the code and license.
I do not have permission to push to libdns/dynv6 though.
Could you maybe take a look?

**Comment by mholt** [comment]
Make sure to accept the invitation. It says "Awaiting sstamoulis’s response"

**Comment by rishubn** [comment]
@mholt Why was this never pushed? libdns/dynv6 is empty

**Comment by mholt** [comment]
@rishubn I dunno, ask @sstamoulis  -- the invite is still pending. :man_shrugging:

**Comment by rishubn** [comment]
Since the code is licensed under MIT, would you object if I submitted a
fork (with correct attributions)? I'd agree to maintain it.

Of course if they decide to return, I will defer to their wishes.

**Comment by mholt** [comment]
Sure, we welcome forks of unmaintained plugins. I'll add you to the repo (and also caddy-dns/dynv6).

**Pull request closed without merging**